test/models/ToolpadRuntime.ts

```diff
@@ -13,10 +13,7 @@ export class ToolpadRuntime {
   }
 
   async gotoPage(pageName: string) {
-    await this.goto();
```

Comment:
I thought that this method was intentionally to test navigation using the in-app UI?
But we can also just have a test for that separately and mainly use this logic if it's more efficient.

Reply:
I don't think we should look at the models this way. They are not tests themselves, they are helpers that abstract common moves so that we have a central place to update when underlying UI elements change.